packages/common/container-definitions/src/runtime.ts

```diff
@@ -198,6 +198,11 @@ export interface IContainerContext {
 	 * @privateremarks Tracking in AB#5714
 	 */
 	readonly id: string;
+
+	/**
+	 * This snapshot contains other artifacts too like blobContents, ops etc.
+	 */
+	readonly snapshot?: ISnapshot;
```

Comment:
I'd probably name it something more to the point on what this represents (`snapshot` property). Type clarifies it to an extent, but when you read in the code, it would be hard to differentiate `baseSnapshot` vs. `snapshot`

Reply:
Changed too: snapshotWithContents